tests/mman/test_mman.c

```diff
@@ -20,6 +21,8 @@
 #define PAGE_SIZE 4096
 #endif
 
+myst_kernel_args_t __myst_kernel_args;
```

Comment:
Is this used anywhere?

Reply:
lol, yes. The test does not build without it as the kernel needs it to be there